nb/pd-pandas-intro.ipynb

```diff
@@ -1953,9 +936,14 @@
     "\n",
     "More often than not real world data is incomplete in some way.\n",
     "In `NumPy`, and therefore in `pandas`, missing data is represented using NaNs (not a number).\n",
-    "NaNs are actuall IEEE 754 float NaNs, therefore the data type of a `Series` (or `NumPy` array)\n",
-    "must be either a float or a Python object.  `Series` strings are just Python objects,\n",
-    "this is contrary to `NumPy`'s arrays; this means that a `series` or a data frame can hold\n",
+    "NaNs used to be IEEE 754 float NaNs, therefore the data type of a `Series` (or `NumPy` array)\n",
+    "should of been either a float or a Python object. Since Pandas version 1.0.0 the NaNs got a\n",
+    "big update and now they can be used inside multiple data types of `Series`, you can read more\n",
+    "about this major upgrade here: https://pandas.pydata.org/docs/user_guide/missing_data.html#missing-data-na.\n",
+    "This is not the case for `NumPy`.\n",
```

Comment:
Nice one, got a rewrite on it.  This thing will likely get profiled and changed a bit in the coming months.